Financely/Financely/settings.py

```diff
@@ -14,6 +14,8 @@
 import os
 
 os.environ["KMP_DUPLICATE_LIB_OK"]="TRUE"
+os.editor["tureureur"] = "asydfasdf;asdf"
```

Comment:
**Issue**: Incorrect usage of os.editor

**Reason**: The code attempts to use `os.editor`, which is not a valid attribute or method in the `os` module.

**Suggestion**: Remove the line `os.editor['tureureur'] = 'asydfasdf;asdf'` as it does not serve any purpose and will cause an AttributeError.